app/controllers/api/v2/hosts_controller.rb

```diff
@@ -38,6 +38,7 @@ def show
       def_param_group :host do
         param :host, Hash, :required => true, :action_aware => true do
           param :name, String, :required => true
+          param :aliases, String, :desc => N_("DNS aliases of the host.")
```

Comment:
I wonder if it makes sense to introduce a object of its own, that is assigned to a domain? e.g.
```ruby
class Alias < ActiveRecord::Base
  belongs_to :domain
  belongs_to :host ?
 ..validate uniq etc etc

end